modules/ROOT/pages/maintenance/space-ids/space-ids.adoc

```diff
@@ -1,84 +1,145 @@
 = Listing Space IDs
 :toc: right
-:description: There are rare situations like when using shell commands that require the ID of a space as parameter. This page shows how to get it.
+:description: There are rare situations like when using shell commands that require the space ID as parameter. This page shows how to get it.
 
 == Introduction
 
 {description} As an example of a use case, see the xref:deployment/services/s-list/storage-users.adoc#manage-trash-bin-items[Manage Trash-Bin Items] command set or the xref:{s-path}/search.adoc#manually-trigger-re-indexing-spaces[Manually Trigger Re-Indexing Spaces].
 
-* Space IDs can currently not be listed by the administrator via the WebUI. They can only be listed via the shell using an authenticated curl GET command accessing the Infinite Scale https://owncloud.dev/apis/[API].
-* End users can enable seeing the WebDAV Urls in their personal preferences of the Infinite Scale WebUI.
+. Via WebDav +
```

Comment:
```suggestion
. Via the WebUI +
For your personal space, open the file view and examine the URL in the browser.
For a manually created project space, open the file view, click on `Spaces`. Click on the desired space and examine the URL.

In either case, the browser URL contains a `fileId` parameter like e.g. this:

`...?fileId=c4dae5a1-7e34-47b2-947c-d9a3283f863f%24847593a2-e3e4-4e5f-ab44-d3d739a8a2ba!847593a2-e3e4-4e5f-ab44-d3d739a8a2ba&...`

The part after the exclamation mark `!` is the space ID.  ( `847593a2-e3e4-4e5f-ab44-d3d739a8a2ba` in this example.) 

. Via WebDav +
```

Reply:
This is not correct. It is for all spaces and you must click on the space and enable the details view. Then you have to look on the `WebDAV Path/URL` to get the spaceID. I will improve the text

Reply:
I have also added a full section below explaining the process. Not necessary to do it here in the overview...